src/components/side_panel/chart/line_chart/line_chart_config_panel.ts

```diff
@@ -7,7 +7,7 @@ export class LineConfigPanel extends LineBarPieConfigPanel {
   get canTreatLabelsAsText() {
     const chart = this.env.model.getters.getChart(this.props.figureId);
     if (chart && chart instanceof LineChart) {
```

Comment:
isn't this completely useless ? If we are in the panel it means that
1. the chart exists
2. it is a LineChart

Reply:
Well 
1) we can never be too careful :D
2) We know that, but typescipt don't. It yells at us without the conditions in the if.